rdr_service/genomic/genomic_job_controller.py

```diff
@@ -112,7 +112,8 @@ def _get_uningested_file_names_from_bucket(self, bucket_name):
         """
         files = list_blobs('/' + bucket_name)
         files = [s.name for s in files
-                 if self.archive_folder_name not in s.name.lower()]
+                 if self.archive_folder_name not in s.name.lower()
+                 if 'datamanifest' in s.name.lower()]
```

Comment:
Is this the same as `if self.archive_folder_name not in s.name.lower() and 'datamanifest' in s.name.lower()` ?

Reply:
Yes, the code does the same thing.